README.md

```diff
@@ -110,6 +113,46 @@ Per-VDOM:
  * `fortigate_certificate_valid_to_seconds`
  * `fortigate_certificate_cmdb_references`
 
+Per-VDOM and Wifi-Client
+ * `fortigate_wifi_client_info`
+ * `fortigate_wifi_client_data_rate_bit_per_second`
```

Comment:
I had to check, but we are using `_bps` for bit_per_second on interface_speed today. Maybe we should keep using that suffix instead of introducing a new one? It isn't clean cut, and I would probably perfer `bit_per_second` given that we have `byte_per_second` - but then again, since _bps is /pretty/ common (especially lower case) to mean bit per second, maybe it's fine to use `_bps` suffix?

What are your thoughts?

Reply:
:thinking: 

What about using right now `bps` and setup a new branch which prepares a v2 and there we change it to `bit_per_second` for all metrics. 
When there are enough changes to make  a major release it worth, we can then merge this branch.

Reply:
Agree. We can do that - or we can just file a "Change to make in v2" issue and we track all these smaller things there, so we don't have to track actual code when it's not worth it.

Reply:
Sure, a meta issue for v2 is also an option